### PR TITLE
docs(installation): Installation page first initialization

### PR DESCRIPTION
**Pull request by educhastenier**
on how to install Bonita Central Docker image

### CODE REVIEW

File: modules/ROOT/pages/installation.adoc [hunk 1]
```diff
@@ -1,3 +1,54 @@
 = Installing Bonita Central 
 :description: Installing Bonita Central
 
+[NOTE]
+====
+For Subscription editions only.
+====
+
+
+Bonita Central is provided as a Docker image that can be downloaded from xref:{bonitaVersion}@bonita::bonita-repository-access.adoc[Bonita Artifact Repository]
```
Comment (educhastenier) [inline modules/ROOT/pages/installation.adoc]:
```suggestion
Bonita Central is provided as a Docker image that can be downloaded from xref:{bonitaVersion}@bonita::bonita-repository-access.adoc[Bonita Artifact Repository]. +
Ensure you have correctly set your credentials up before going any further.
```

File: modules/ROOT/pages/installation.adoc [hunk 2]
```diff
@@ -2,4 +2,56 @@
...
+
+where:
+
+* `<HOSTNAME>` is the name of the machine for which you have generated the Bonita license file
+* `<CENTRAL_LIC_FOLDER>` is the folder where you place the license file
+
+
+== Optional docker configuration
+
+**_FIXME_**: _Do we want to document that_ ?!!!!
```
Comment (jeromecambon) [inline modules/ROOT/pages/installation.adoc]: I guess we should remove this. This is anyway surprising to expose it...

Comment (pabloalonso) [reply modules/ROOT/pages/installation.adoc]: We should not document neither expose it

File: modules/ROOT/pages/installation.adoc [hunk 3]
```diff
@@ -2,4 +2,56 @@
...
+
+
+Bonita Central is provided as a Docker image that can be downloaded from xref:{bonitaVersion}@bonita::bonita-repository-access.adoc[Bonita Artifact Repository]. +
+Ensure you have correctly set your credentials up before going any further.
+
+
+== Retrieving the Docker image
+
+```bash
+docker image pull bonitasoft.jfrog.io/docker/bonita-central:1.0.0
```
Comment (pabloalonso) [inline modules/ROOT/pages/installation.adoc]: bonitasoft.jfrog.io/docker-releases/bonita-central:1.0.0-Beta-01

**Review by jeromecambon — APPROVED** [review]

**Pull request merged**